index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+/* eslint-disable no-console */
+
+class Executor {
```

Comment:
I added the fields to the comments. We can add validation during hip week for this and also the datastore-base?